debian/rules

```diff
@@ -12,7 +12,16 @@ source=$(CURDIR)/../server
 
 # main packaging script based on dh7 syntax
 %:
-	dh $@ 
+	dh $@ --with=systemd
```

Comment:
what does adding this arg do?

Reply:
It triggers _debhelper_ into adding the necessary boiler plate to the install scripts to install the systemd unit file, and enable the unit. Explained here:

> The method used is that published in the Debian systemd packaging wiki:
> https://wiki.debian.org/Teams/pkg-systemd/Packaging